docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -0,0 +1,53 @@
+.. _2017_risk_exposure_template:
+
+----------------------
+Risk Exposure Template
+----------------------
+
+.. todo::
+  Add instructions, following Nathaniel's cause model template instructions. Flesh out all headings.
+
+
+Risk Exposure Overview
+----------------------
+
+ - expert opinion
+
+Risk Exposure description
+-------------------------
+
+Risk Exposures Description in GBD
+------------------------------
+
+  - Type of risk (categorical, continuous, etc.)
+
+  - What is this a risk for?

```

Comment:
Maybe we could rephrase "what is this a risk for?" to "what causes are affected by this risk?" for clarity, but it might not need to be done for this PR.